spanner/integration_test.go

```diff
@@ -623,7 +623,7 @@ func TestIntegration_SingleUse(t *testing.T) {
 		{
 			name: "max_staleness",
 			want: [][]interface{}{{int64(1), "Marc", "Foo"}, {int64(3), "Alpha", "Beta"}, {int64(4), "Last", "End"}},
-			tb:   MaxStaleness(time.Second),
+			tb:   MaxStaleness(time.Nanosecond),
 			checkTs: func(ts time.Time) error {
 				if ts.Before(writes[3].ts) {
```

Comment:
```suggestion
				if writes[3].ts.Sub(ts) < time.Second {
```

Reply:
Can we do this, instead of reducing the max staleness, so that the check is more undertandable?

Reply:
This case will always fail because the difference between 2 timestamps will always be less than 1 sec

Reply:
Updated